### DISCUSSION TITLE
Use mantine 7.3.2  and get an error

### DISCUSSION THREAD

**Discussion by leah-committed** [mantinedev/mantine#5439] [Q&A]
Error: Build failed with 21 errors:
node_modules/@medplum/react/dist/esm/index.mjs:1:24878: ERROR: No matching export in "node_modules/@mantine/core/esm/index.mjs" for import "createStyles"        
node_modules/@medplum/react/dist/esm/index.mjs:1:24909: ERROR: No matching export in "node_modules/@mantine/core/esm/index.mjs" for import "Header"
node_modules/@medplum/react/dist/esm/index.mjs:1:26563: ERROR: No matching export in "node_modules/@mantine/core/esm/index.mjs" for import "createStyles"        
node_modules/@medplum/react/dist/esm/index.mjs:64:3238: ERROR: No matching export in "node_modules/@mantine/core/esm/index.mjs" for import "createStyles"        
node_modules/@medplum/react/dist/esm/index.mjs:64:3257: ERROR: No matching export in "node_modules/@mantine/core/esm/index.mjs" for import "Navbar"

longing for help
Thank you!

**Comment by rtivital** [comment]
The error says that you are using Header, Navbar and createStyles components which are not available in v7

**Comment by leah-committed** [comment]
Thanks for the comment
The question is, are there alternative ways?



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
18.12.23,
13:20:24

‫בתאריך יום ב׳, 18 בדצמ׳ 2023 ב-13:17 מאת ‪Vitaly Rtishchev‬‏ <‪
***@***.***‬‏>:‬

**Reply by rtivital** [reply]
You can find a guide for migration here – https://mantine.dev/changelog/7-0-0/
